docs/README.md

```diff
@@ -12,9 +12,10 @@ organized into a Docusaurus site for easy consumption.
 [Docusaurus](https://docusaurus.io/) is a framework for static site generation,
 focusing on documentation-based sites.
 
-The `sidebars.json` and `siteConfig.js` are the main entrypoints for updating
-our documentation site. The former determines which README docs are displayed,
-and the latter sets site-wide config values.
+The [`sidebars.json`](https://v1.docusaurus.io/docs/en/navigation) and
+[`siteConfig.js`](https://v1.docusaurus.io/docs/en/site-config) are the main
+entrypoints for updating our documentation site. The former determines which
+README docs are displayed, and the latter sets site-wide config values.
```

Comment:
minor nit: Can you also add the naming convention around the what the ID value should be for each document.